src/bisect.cpp

```diff
@@ -105,7 +126,7 @@ std::vector<long> bisect_multilevel(bulk::world& world,
         C_list[0].merge_free_vertices(HC_list[0]);
     }
 
-    auto time = bulk::util::timer();
+    //auto time = bulk::util::timer();

```

Comment:
The timer was placed here because then it only times the actual partitioning, do we want to move it back?

Reply:
Yes, this may lead to better anaylitics of the performance